python/azureml/hybrid.py

```diff
@@ -0,0 +1,316 @@
+# ------------------------------------------------------------------------------------------------------
+# The file implemented three ways to merge a local model and a azure-proxy model for hybrid inferencing:
```

Comment:
Someone new to this file is going to ask what is a azure-proxy model.

Reply:
Explaining this in README.md